src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11351,8 +11316,6 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
         // Evaporator (IU side) operational parameters
         Pevap = this->refrig->getSatPressure(state, this->IUEvaporatingTemp, RoutineName);
         Psuction = Pevap;
-        Tsuction = this->IUEvaporatingTemp; // GetSatTemperatureRefrig(state,  this->RefrigerantName, max( min( Psuction, RefPHigh ), RefPLow ),
-                                            // RefrigerantIndex, RoutineName );
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:11444]:(style),[redundantAssignment],Variable 'Tsuction' is reassigned a value before the old one has been used.